tasks/main.yml

```diff
@@ -20,3 +20,9 @@
     - ansible_os_family == "FreeBSD"
     - item.rc != 0
   with_items: "{{ register_virtualbox_kldstat.results }}"
+
+- name: Enable vboxnet service
+  service:
+    name: vboxnet
+    enabled: yes
+  when: ansible_os_family == "FreeBSD"
```

Comment:
if anything specific to a platform must be managed in `main.yml`, i prefer `include` `configure-{{ ansible_os_family }}.yml`. main.yml should focus on the package, not platforms.